<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CharacterSkillTreeItemNameAndSkillPointsWidget</class>
 <widget class="QWidget" name="CharacterSkillTreeItemNameAndSkillPointsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>194</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QLabel" name="skillBookPimapLabel">
     <property name="text">
      <string>Skill Book Pixmap Here</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>1</number>
     </property>
     <item>
      <widget class="QLabel" name="skillNameLabel">
       <property name="text">
        <string>Skill Name Here</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="skillPointsLabel">
       <property name="text">
        <string>Skill Points Here</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
